src/model/exercise.coffee

```diff
@@ -20,21 +26,22 @@ Exercises =
       url += "&entry.1091629000=#{BOOK_UID_XREF[options.bookUUID]}"
     url
 
-
-  getStimulus: (exercise) ->
-    exercise.content?.stimulus_html or ''
-
   getParts: (exercise) ->
     exercise.content?.questions or []
 
   isMultipart: (exercise) ->
     @getParts(exercise).length > 1
 
   hasInteractive: (exercise) ->
-    !!@getStimulus(exercise).match(/iframe.*(cnx.org|phet.colorado.edu)/)
+    exercise.has_interactive
 
   hasVideo: (exercise) ->
-    !!@getStimulus(exercise).match(/(youtube.com\/(v|embed)\/|khanacademy.org)/)
+    exercise.has_video
+
+  replacePlaceholders: (content) ->
```

Comment:
yeah, I was planning on doing so, but then realized the placeholder isn't really a SVG, they're components that are rendered.  So then we need to render the strings before/after and then the image in place.  I still like the idea of the div, and have future plans for how we can actually replace them with CSS.